packages/components/src/molecules/Table/Table.tsx

```diff
@@ -6,17 +6,32 @@ export interface TableProps extends DetailedHTMLProps<TableHTMLAttributes<HTMLTa
    * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
    */
   testId?: string
+  /**
+   * Children for Table components.
+   */
   children: React.ReactNode
+  /**
+   * Defines what style this component should use.
+   */
+  variant?: 'colored' | 'bordered'
 }
 
 const Table = forwardRef<HTMLTableElement, TableProps>(function Table(
-  { testId = 'store-table', children, ...otherProps },
+  { variant = 'colored', testId = 'fs-table', children, ...otherProps },
   ref
 ) {
   return (
-    <table ref={ref} data-fs-table data-testid={testId} {...otherProps}>
-      {children}
-    </table>
+    <div data-fs-table>
```

Comment:
do we really need this div here as a wrapper? 👀

Reply:
yep, in order to add horizontal scroll on wide tables 🙃